oasis-webapp/src/main/resources/templates/Login.soy

```diff
@@ -173,10 +175,9 @@ aside a { display: inline-block; padding: 5px 15px; font-weight: lighter; text-d
   <aside class=two-cols>
     <span style="height: 0px;"></span>
     <span>
-      {let $overview kind="text"}{msg desc="overview URL"}https://www.ozwillo.com/en/oz/discover{/msg}{/let}
+      {let $overview kind="text"}{msg desc="overview URL"}https://www.ozwillo.com/en/discover{/msg}{/let}
```

Comment:
I've just deployed a new version of the website with support for */oz* URLs so no need to change them on your side.

URLs in other languages are already mapped, although content will be displayed in English.